recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md

```diff
@@ -0,0 +1,70 @@
+    4. You know when to accept technical debt for the sake of the customer and you pay it down at the right moment. You honor organizational commitments made to the customer.
+    5. You escalate product requirements when they are unclear or not in the _overall _best interest of your customers/users.
+4. **_Leadership: sometimes engineering is more about people than about writing code…_**
+    1. You create positive pressure to ship. You help engineers see the _why_ so they put in maximum energy on the _what_ and the _how_. You have an intensity that is hard to describe and it energizes the people around you.
+    2. You know how to communicate perfectly: the message, the medium, the audience, the timing. Your stakeholders always know the project's status and tradeoffs chosen. Your partners/dependencies always know what you need from them and keep your requirements top-of-mind. Your leadership chain always knows the project's risks and how they can help accelerate you, if they so choose.
+    3. You keep the wheels greased. You have healthy relationships with all your partner teams, Tech Leads, dependencies, etc. Your project is never delayed because you get incorrectly blocked by external factors.
+    4. You escalate when appropriate: bad technical decisions, poor work quality, inappropriate workload, unreasonable technical debt, etc. 
+    5. You regularly discuss flaws/friction with Tech Leads based on your experience working in their systems. They always have your needs top-of-mind when making design and prioritization decisions. You drive improvements in the overall technical stack through your unique combination of product awareness, people leadership, and technical strength.
+5. **_Inspirational: you get your engineering team excited about the work_**
+    1. You live and breathe your product. Your enthusiasm is infectious and you spread it actively through demos, celebrations, customer stories, etc.
+    2. You are a talent magnet. People seek you out to work with you, your team, and specifically the feature you are building. You understand that talent comes in many shapes and sizes and you know how to leverage them all: 20% time, sub-delegation to another person/team, temporary allocation, or even permanent moves. In all cases, you help your recruits become part of the team.
+    3. You build a team of owners, not contractors. Your engineers care about the product they are building. Your team regularly ships ideas that came _from the team_. 
+    4. You build a [Level 4 Tribe](https://en.wikipedia.org/wiki/Tribal_Leadership): the team has fun, loves what they do, and may be friends forever. 
+    5. You don’t burn yourself or your team out. You can keep an impressive pace for years.
```

Comment:
This seems really low down the list... I wonder if it should not be at least mentioned somewhere above as well, where the impressive deliveries on tight deadlines live...

Reply:
Nitpicking :)